wagtail/contrib/settings/templates/wagtailsettings/edit.html

```diff
@@ -15,7 +15,8 @@
             </form>
         {% endif %}
     {% endfragment %}
-    {% include "wagtailadmin/shared/header.html" with title="Editing" icon="cogs" subtitle=setting_type_name|capfirst merged=1 extra_actions=editing_actions %}
+    {% trans "Editing" as editing_str %}
+    {% include "wagtailadmin/shared/header.html" with title=editing_str icon="cogs" subtitle=setting_type_name|capfirst merged=1 extra_actions=editing_actions %}
```

Comment:
note - I can see here we are already using `|capfirst` but for a similar purpose.